src/components/tables/AccessGrants/AccessLinks/Dialog/GenerateInvitation.tsx

```diff
@@ -18,6 +19,8 @@ interface Props {
     setServerError: React.Dispatch<React.SetStateAction<PostgrestError | null>>;
 }
 
+const namePattern = new RegExp(`^[a-zA-Z0-9-_./]+$`);
```

Comment:
Please replace with PREFIX_NAME_PATTERN from misc utils

Reply:
This is especially important because I believe this pattern is actually incorrect. This would allow "acemeCo/foo////////" and that is not allowed/valid.

Reply:
We chatted about this outside this thread. The regular expression needs to be an extension of that defined by `PREFIX_NAME_PATTERN` because these prefixes should always end in a slash. The highlighted expression here is incorrect. I will update it so that is checks for a single, concluding forward slash.